config/database.yml

```diff
@@ -2,6 +2,8 @@ default: &default
   adapter: postgresql
   pool: 5
   timeout: 5000
+  username: <%= ENV['DATABASE_USER'] %>
+  host: <%= ENV['DATABASE_HOST'] %>
```

Comment:
If we're adding env variables, we should do it consistently across dev environments.  

This project uses the dotenv gem to manage env variables in preprod environments.  We haven't had a chance to actually use it until now.  

There are setup instructions [here](https://github.com/bkeepers/dotenv).

It'd be great if you could add a `.env.sample` file in the root directory and then update the setup script to copy that file over to `.env.`  (`.env` should be added to .gitignore).   The `.env.sample` file should contain sensitive variables like `DATABASE_USER` and `DATABASE_HOST`.